examples/pages/react/TreeMenu.tsx

```diff
@@ -176,6 +176,7 @@ export class TreeMenuDocs extends React.Component<{}, {}> {
                     <Markup.ReactMarkupPreview>
                         <div className='docs-page__content-row docs-page__content-row--no-margin'>
                             <TreeMenu
+                                searchPlaceholder='Search...'
```

Comment:
We can't use english strings that would be visible to users. That's because translations wouldn't work then. Instead take placeholder as a prop.

Reply:
but this is just example page

Reply:
good point, I didn't see this